.prettierignore

```diff
@@ -0,0 +1,11 @@
+.circleci/
```

Comment:
I think it would be nice, if we only ignore circleci/config.yml
Why not format the other files with prettier? (Tested it and works fine, output config,yml is the same for formatted src and not formatted src.